less/_variables.less

```diff
@@ -191,7 +157,7 @@
 @input-height: @btn-height;
 @input-reset-bgcolor: transparent;
 @input-reset-bordercolor: @searchbar-reset-bordercolor;
-@input-disabled-bordercolor: @btn-disabled-bordercolor;
+@input-disabled-bordercolor: @btn-disabled-border;
```

Comment:
Seems like a weird change to make when the others consistently end in "color". I'll get over the difference, mostly just wanted to know why.